server/config/main.go

```diff
@@ -37,7 +36,8 @@ const (
 	// the date changed between 23:59 and 00:00:xx2.
 	RunnerInterval = 25 * time.Second
 
-	OverrideUsername = "Raven"
+	BotUsername = "raven"
```

Comment:
What appears as the name besides the avatar depends on individual's setting (`Teamate Name Display`), can we do something to make sure the name always appears as `Raven` and not as `raven`?

Reply:
I have investigated on this but didn't find any way to do this as we can't use upper case in username.